src/devtools/client/debugger/src/components/PrimaryPanes/Sources.css

```diff
@@ -188,66 +161,3 @@
   flex: 1;
   overflow: auto;
 }
-
-.source-outline-panel.has-root > div {
-  height: 100%;
-}
-
-.source-outline-panel.has-root .thread-header {
-  margin-top: 4px;
-}
-
-.sources-list .img.blackBox {
```

Comment:
Do we not have the blackbox feature anymore?

Reply:
good call we should keep this